### DISCUSSION TITLE
Accessing `useAppConfig()` in `server/api/` dir

### DISCUSSION THREAD

**Discussion by ijkml** [nuxt/nuxt#19180] [Questions] [answered]
[According to the docs here](https://nuxt.com/docs/guide/directory-structure/app-config#defining-app-config), app config can be used in both server and client, but when I try to access `useAppConfig()` in `server/api/[file]` I get an error: `useAppConfig is not defined`. The editor shows correct types, though. Please, what am I missing? :man_shrugging:

```ts
export default defineEventHandler(async (event) => {
  const config = useAppConfig();
  console.info(config);
  return 'success';
});
```

**Comment by ndragun92** [comment, marked as answer]
You can access it from server side/client side (when SSR is enabled) in your app, not in server/api folder. I think that was a misunderstanding but someone can correct me.

**Reply by TheAlexLichter** [reply]
Correct. Right now, it is not possible. Will be improved in the future though in a way or another.

**Reply by ijkml** [reply]
I had the same thought initially, but that particular line in the docs piqued my interest. Thank you.

**Reply by TheAlexLichter** [reply]
PR welcome! Server means in this case "Vue components during SSR", not the "Nitro" part.

**Reply by ijkml** [reply]
Done! #19200
